crates/blockifier/src/execution/native_syscall_handler.rs

```diff
@@ -47,6 +54,40 @@ pub struct NativeSyscallHandler<'state> {
     pub events: Vec<OrderedEvent>,
     pub l2_to_l1_messages: Vec<OrderedL2ToL1Message>,
     pub inner_calls: Vec<CallInfo>,
+    // Secp hint processors.
+    pub secp256k1_hint_processor: SecpHintProcessor<ark_secp256k1::Config>,
+    pub secp256r1_hint_processor: SecpHintProcessor<ark_secp256r1::Config>,
+}
+impl NativeSyscallHandler<'_> {
+    fn allocate_point<Curve: SWCurveConfig>(
+        point_x: U256,
+        point_y: U256,
+        hint_processor: &mut SecpHintProcessor<Curve>,
+    ) -> SyscallResult<usize>
+    where
+        Curve::BaseField: PrimeField,
+    {
+        let request = SecpNewRequest { x: u256_to_biguint(point_x), y: u256_to_biguint(point_y) };
+
+        let response = hint_processor.secp_new_unchecked(request);
+
+        Self::parse_allocate_point_response(response)
+    }
+
```

Comment:
I don´t think this should be a method for NativeSyscallHandler, it is more of a utility